index.html

```diff
@@ -0,0 +1,447 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="icon" href="./favicon.ico" />
+    <title>Aziz Mejri - Portfolio</title>
+  </head>
+  <body>
+    <nav>
+      <div id="nav-bar">
+        <div class="nav-cont container header">
+          <a id="logo" href="#">Aziz Mejri</a>
+          <button type="submit" class="menu-bars">
+            <i class="fas fa-bars"></i>
+          </button>
+          <ul id="nav-list" class="">
```

Comment:
According to HTML & CSS Best Practices, it is not a good practice to include empty attributes in your code. Please remove the `class` attribute as it is of no use. This helps in having clean and readable code.